gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -1118,9 +1117,11 @@ func (vm *VMKeeper) QueryStorage(ctx sdk.Context, pkgPath string) (string, error
 
 func (vm *VMKeeper) processStorageDeposit(ctx sdk.Context, caller crypto.Address, deposit std.Coins, gnostore gno.Store, params Params) error {
 	realmDiffs := gnostore.RealmStorageDiffs()
-	depositAmt := deposit.AmountOf(ugnot.Denom)
```

Comment:
The storage deposit should use ugnot.Denom instead of deriving it from DefaultDeposit’s denom, which is modifiable as a parameter.